<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_0">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>220</width>
     <height>600</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string>Approximation</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string>A</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string>B</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string>a</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string>Rotation</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string>X shift</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string>Y shift</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
